rakelib/utils.rake

```diff
@@ -29,4 +46,33 @@ class String
     return false if self.empty? || self =~ (/^(false|f|no|n|0)$/i)
     raise ArgumentError.new "invalid value: #{self}"
   end
+
+  # colorization
+  def black;          colors_available? ? "\e[30m#{self}\e[0m" : self end
+  def red;            colors_available? ? "\e[31m#{self}\e[0m" : self end
+  def green;          colors_available? ? "\e[32m#{self}\e[0m" : self end
+  def brown;          colors_available? ? "\e[33m#{self}\e[0m" : self end
+  def blue;           colors_available? ? "\e[34m#{self}\e[0m" : self end
+  def magenta;        colors_available? ? "\e[35m#{self}\e[0m" : self end
+  def cyan;           colors_available? ? "\e[36m#{self}\e[0m" : self end
+  def gray;           colors_available? ? "\e[37m#{self}\e[0m" : self end
+
+  def bg_black;       "\e[40m#{self}\e[0m" end
+  def bg_red;         "\e[41m#{self}\e[0m" end
+  def bg_green;       "\e[42m#{self}\e[0m" end
+  def bg_brown;       "\e[43m#{self}\e[0m" end
+  def bg_blue;        "\e[44m#{self}\e[0m" end
+  def bg_magenta;     "\e[45m#{self}\e[0m" end
+  def bg_cyan;        "\e[46m#{self}\e[0m" end
+  def bg_gray;        "\e[47m#{self}\e[0m" end
+
+  def bold;           "\e[1m#{self}\e[22m" end
```

Comment:
Yeah thought about that too, but it's too easy to add that I don't think it's worth it for a couple of lines and usages for us?